### ISSUE TITLE
Unable to change administrator user settings

### ISSUE THREAD

**Issue by maartenvn** [directus/next#430]
**Directus**: 9.0.0-beta.2
**Node**: v14.10.0
**MySQL**: Ver 14.14

I am unable to change the administrator user language from English to Dutch:

![image](https://user-images.githubusercontent.com/6013068/94453398-e737d580-01b0-11eb-85a5-d69f4de5f110.png)

```json
{
	"errors": [
		{
			"message": "update `directus_users` set `language` = 'nl-NL', `last_access` = '2020-09-28T05:35:15+02:00' where `id` in ('<UUID HERE>') - ER_TRUNCATED_WRONG_VALUE: Incorrect datetime value: '2020-09-28T05:35:15+02:00' for column 'last_access' at row 1",
			"extensions": {
				"code": "INTERNAL_SERVER_ERROR"
			}
		}
	]
}
```

The error from the API is also not propery handled in the frontend. I get this message when I try to save:
![image](https://user-images.githubusercontent.com/6013068/94453496-06cefe00-01b1-11eb-87fe-e58d0d11fedb.png)

**Comment by maartenvn** [comment]
This also occurs for other fields on the user settings page

**Comment by rijkvanzanten** [comment]
I think this was resolved in #438. We can reopen if not
